commit a4ac0d847af9dd34d5953a5e264400326144b6b2
Author: Henry Ptasinski <henryp@broadcom.com>
Date:   Thu Oct 21 11:37:53 2010 -0700

    staging: brcm80211: brcmfmac: Removed codeversion from firmware filenames.
    
    The basename-apiversion-codeversion construction for firmware filenames is not
    used by most other firmware files, adds complexity, and is not providing any
    value.  Rename the firmware files using just basename-apiversion.
    
    Users of the brcmfmac driver will need to update the linux-firmware package
    to get the renamed files, or manually rename their installed firmware files.
    
    Signed-off-by: Henry Ptasinski <henryp@broadcom.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

--- a/drivers/staging/brcm80211/brcmfmac/wl_cfg80211.c
+++ b/drivers/staging/brcm80211/brcmfmac/wl_cfg80211.c
@@ -52,8 +52,8 @@ static struct wl_dev *wl_cfg80211_dev;
 
 u32 wl_dbg_level = WL_DBG_ERR | WL_DBG_INFO;
 
-#define WL_4329_FW_FILE "brcm/bcm4329-fullmac-4-218-248-5.bin"
-#define WL_4329_NVRAM_FILE "brcm/bcm4329-fullmac-4-218-248-5.txt"
+#define WL_4329_FW_FILE "brcm/bcm4329-fullmac-4.bin"
+#define WL_4329_NVRAM_FILE "brcm/bcm4329-fullmac-4.txt"
 
 /*
 ** cfg80211_ops api/callback list
